docs/source/tutorials/getting-started/intro.rst

```diff
@@ -27,15 +28,35 @@ executing the following command:
    ./get-dependencies.sh
    daml studio
 
-Daml Finance Reference App
-**************************
+Reference App
+*************
 
 In addition to Daml Finance, there also a separate Daml Finance Reference App.
 It showcases several of the Daml Finance capabilites in a web-based graphical user interface.
 
 If you are interested in trying out the Daml Finance Reference App locally, you can clone the corresponding repo
 and follow the installation instructions on the `Daml Finance Reference App GitHub page <https://github.com/digital-asset/daml-finance-app>`_.
 
+
+Create your own App
+*******************
+
+It is possible to generate *JavaScript* code from the Daml Finance packages you need, by running
```

Comment:
I think it makes sense to link to the pages in the docs for the bindings, e.g.
- https://docs.daml.com/app-dev/bindings-ts/daml2js.html
- https://docs.daml.com/app-dev/bindings-java/index.html

Reply:
Good point, I have added the links.